src/App.js

```diff
@@ -1,7 +1,7 @@
 import './App.css';
 import ApplicationForm from './components/CreateListing';
 import LoginPage from './pages/LoginPage';
-import Signup from './components/Signup';
+import Signup from './components/Signin';
```

Comment:
Is this used anywhere in this file? Can we just remove entirely?

```suggestion
import Signin from './components/Signin';
```

Reply:
Good catch. It was used in LoginPage.js, wasn't used in this App.js file.